hacep/src/main/java/org/kie/hacep/EnvConfig.java

```diff
@@ -28,6 +28,7 @@
     private String snapshotTopicName;
     private String kieSessionInfosTopicName;
     private String printerType;
+    private Integer iterationBetweenSnapshot = 10;
```

Comment:
Use constant DEFAULT_ITERATION_BETWEEN_SNAPSHOT instead of hard coded number?

Reply:
Ok